plugins/database/cassandra/connection_producer.go

```diff
@@ -269,6 +230,50 @@ func (c *cassandraConnectionProducer) createSession(ctx context.Context) (*gocql
 	return session, nil
 }
 
+func getSslOpts(certBundle *certutil.CertBundle, minTLSVersion string, insecureSkipVerify bool) (*gocql.SslOptions, error) {
+	var tlsConfig *tls.Config
```

Comment:
Can we start off with `tlsConfig := &tls.Config{}` if this is never expected to be nil?

Reply:
Funnily enough we just fixed this in another DB: https://github.com/hashicorp/vault/pull/10899/files

Reply:
Yes. Fixed.